packages/console/src/pages/Users/index.tsx

```diff
@@ -102,6 +107,18 @@ const Users = () => {
           ))}
         </tbody>
       </table>
+      {/* TODO: LOG-1977 pagination component */}
+      {data?.[1] && (
+        <div>
+          current page: {pageIndex}, total page: {Math.ceil(data[1] / pageSize)},{' '}
+          <Button
+            title="admin_console.connectors.next"
+            onClick={() => {
+              setPageIndex((previous) => previous + 1);
+            }}
+          />
```

Comment:
Should hide this button when it's already the last page. Also, where's the `Previous` button?

Reply:
same as above